src/Raven.Embedded/RavenEmbedded.cs

```diff
@@ -35,6 +39,29 @@ public void StartServer(ServerOptions options = null)
             if (Interlocked.CompareExchange(ref _serverTask, startServer, null) != null)
                 throw new InvalidOperationException("The server was already started");
 
+            if (options.Security != null)
+            {
+                _certificate = options.Security.ClientCertificate;
+
+                try
+                {
+                    var thumbprint = options.Security.ServerCertificiateThumbprint;
+                    RequestExecutor.ServerCertificateCustomValidationCallback +=
+                        (message, certificate2, chain, errors) =>
+                            certificate2.Thumbprint == thumbprint;
```

Comment:
Shoud be configurable. What if we have several thumbprints etc? ServerCertificateCustomValidationCallback is static

Reply:
Not sure if this is something that we need to support with embedded? Only one instance of server will be started by application.

Reply:
The server still needs to be trusted

Reply:
And this add trust to the server.
User can also configure this separately from us, which will overwrite this logic.